src/containers/ServiceList/ServiceList.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {Grid, Card, CardContent, CardMedia, Typography, withStyles} from '@material-ui/core';
+
+const styles = {
+	root: {
+		width: '100%',
+	},
+	card: {
+		height: '20em',
+		cursor:'pointer'
+	},
+	CardContent: {
+		height: '20%',
+	},
+	media: {
+		height: '70%',
+		objectFit: 'fill'
+	}
+};
+
+function ServiceList(props) {
+	const classes = props.classes;
+	const columnWidth = 12/props.servicesPerLine;
+	return (
+		<> 
+			<Grid container spacing={3} className={classes.root}>
+				{	
+					props.services ? 
+						props.services.map( (service, i) => {
+							return <Grid item xs={12} sm={columnWidth} key={i}> 
+								<Card
+									className={classes.card}
+									onClick={() => props.onClick(service.id)}
+								>
+									<CardMedia 
+										className={classes.media}
+										component='img'
+										image='https://picsum.photos/id/840/200/500'
```

Comment:
Det må gerne være et random for at kunne se forskel på dem - plus, smid det gerne ud i en variable et sted

Reply:
tænkte bare at det var lidt ligegyldigt eftersom at det var temporary og skulle ændres når services havde billeder på sig. Men kan godt lave det random.